brody_lab_to_nwb/column_mapping.csv

```diff
@@ -0,0 +1,16 @@
+mat_name,nwb_name,nwb_description
+c_poke,c_poke_time,
+hit_hist,hit_history,"If the trial was a hit or miss."
+trial_num,trial_number,"Original index of the trial."
```

Comment:
Can this be the id of the row of the trials table?

Reply:
`trial_num`? Not necessarily, sometimes they want to clip some or throw some out so they may not correspond to a strict index

Reply:
ok